From 84d45ee2c76316b7a31d61be06d2c919c1cc06fc Mon Sep 17 00:00:00 2001
From: Josua Mayer <josua@solid-run.com>
Date: Sun, 1 Oct 2023 13:53:05 +0200
Subject: [PATCH] arm64: dts: hummingboard-t: update m.2 default pci interface

As PCI is more often requested, and does not physically collide with the
current heatsink, change default to PCI.

Signed-off-by: Josua Mayer <josua@solid-run.com>
---
 arch/arm64/boot/dts/ti/k3-am642-hummingboard-t.dts | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/ti/k3-am642-hummingboard-t.dts b/arch/arm64/boot/dts/ti/k3-am642-hummingboard-t.dts
index d579f176c717..fd92082afcf2 100644
--- a/arch/arm64/boot/dts/ti/k3-am642-hummingboard-t.dts
+++ b/arch/arm64/boot/dts/ti/k3-am642-hummingboard-t.dts
@@ -20,7 +20,7 @@
  * - PHY_TYPE_PCIE
  * - PHY_TYPE_USB3
  */
-#define SERDES0_PHY_TYPE PHY_TYPE_USB3
+#define SERDES0_PHY_TYPE PHY_TYPE_PCIE
 
 / {
 	model = "SolidRun AM6442 HummingBoard-T";
-- 
2.35.3

